test/lib/agent_helper.js

```diff
@@ -246,9 +246,9 @@ const helper = module.exports = {
    * @param {function} callback
    *  The operations to be performed while the server is running.
    */
-  bootstrapRedis: (redis, dbIndex, callback) => {
+  flushRedisDb: (redis, dbIndex, callback) => {
```

Comment:
Since this basically did like one thing tangibly (although I do think the selection of the index persists), I updated the function name to be clear it is flushing the redis DB.